### ISSUE TITLE
About warnings.

### ISSUE THREAD

**Issue by leeyunhome** [sid0312/anpr_yolov5#2]
Hello,

(yolov5_env) D:\coding\GitHub\anpr_yolov5>python detect.py --source test_cars/  --weights weights/last.pt --conf 0.4
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.4, device='', fourcc='mp4v', img_size=640, iou_thres=0.5, output='inference/output', save_txt=False, source='test_cars/', view_img=False, weights='weights/last.pt')
Using CPU

C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.container.Sequential' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.conv.Conv2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.batchnorm.BatchNorm2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.activation.LeakyReLU' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.container.ModuleList' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.pooling.MaxPool2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
C:\Users\mhbang\miniconda3\envs\yolov5_env\lib\site-packages\torch\serialization.py:649: SourceChangeWarning: source code of class 'torch.nn.modules.upsampling.Upsample' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
image 1/100 test_cars\car_old_0001.jpg: 512x640 1 license_plates, Done. (0.555s)
image 2/100 test_cars\car_old_0002.jpg: 512x640 1 license_plates, Done. (0.783s)

Thank you.

**Comment by sid0312** [comment]
It looks like the image (which is being normalized) is on the GPU, whereas the mean and standard deviation tensors are on the CPU. We're supplying the mean and standard deviations as lists, so they're converted to tensors by PyTorch internally. It's weird – I don't get the error when I try it on my computer. You could try putting the image on the GPU after the normalization:
(You can ignore the other warnings, they're only because some layer definitions were changed after the model was trained.)

**Comment by sid0312** [comment]
this may be mostly due to an upgraded version of pytorch. downgrade your pytorch version

**Comment by leeyunhome** [comment]
By the way, since it is a warning, is there any problem if I use it as it is?
If there is a problem, where would it be?

Thank you.

**Comment by sid0312** [comment]
there are discrepancies with the torch.nn module (new patches have been made).
just use 
`torch.nn.Module.dump_patches = True` after importing torch
and you'll be fine
